### PR TITLE
phase5 jenkins

### PR DESCRIPTION
**Pull request by aoware1**

### CODE REVIEW

File: jenkins-master-script.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        mysql-client 
+        unzip 
+        tree 
+        default-jdk
+        default-jre
+        maven
+        gnupg
+        software-properties-common
+    )
+
```
Comment (devopseasylearning) [inline jenkins-master-script.sh:51]: there are  lot pf unnecessary pkgs above

**Review by devopseasylearning — APPROVED** [review]
looks good

**Review by s6ronald — APPROVED** [review]

**Review by confidencedenku — APPROVED** [review]
Everything looks good to me. kudos to the team of engineers!

**Review by s6charlsee — COMMENTED** [review]
good work

**Pull request merged**